src/components/keyboard-nav.js

```diff
@@ -0,0 +1,97 @@
+import _ from 'lodash/fp'
+import { useRef } from 'react'
+import { div, h } from 'react-hyperscript-helpers'
+
+
+export const ArrowKeyNavigation = ({ keyDownHandler, children, ...props }) => {
+  const ref = useRef()
+
+  // Figure out which child elements are focusable/clickable
+  const focusableElements = () => {
+    return ref.current.querySelectorAll('a, button, input, select, textarea, [tabindex]')
+  }
+
+  // Figure out which focusable element sent the event
+  const findIndex = event => _.findIndex(f => f === event.target, focusableElements())
```

Comment:
I _think_ you can just use the shorthand for matching:
```suggestion
  const findIndex = event => _.findIndex(event.target, focusableElements())
```

Reply:
Unfortunately that doesn't seem to work for some reason.  I just tried it again and it always returns `0` (but not `-1`, odd...)

Maybe lodash does something weird because it's an object. All I need here is referential equality.